sirepo/status.py

```diff
@@ -99,3 +101,11 @@ def _run_tests():
             uri_router.call_api('runCancel', data=d)
         except Exception:
             pass
+
+
+def _validate_auth_state():
+    r = pkcompat.from_bytes(uri_router.call_api('authState').data)
+    m = re.search(r'SIREPO.authState = (.*?);', r)
```

Comment:
make the regex a bit looser: `r'SIREPO.authState\s*=\s*(.*?);'`. I know it's picky, but it won't break if we ever change the format.

Reply:
8e5eee4e